src/platform/forms/validations.js

```diff
@@ -348,6 +348,14 @@ function validateLength(
   };
 }
 
+function validateWhiteSpace(errors, input) {
```

Comment:
I was trying to find a better name for this, but came up with nothing. :man_shrugging:

Can you add a little comment above this saying that it ensures input isn't _only_ whitespace?